packages/xo-web/src/xo-app/vm-import/index.js

```diff
@@ -192,13 +196,17 @@ const getRedirectionUrl = vms =>
   vms.length === 0
     ? undefined // no redirect
     : vms.length === 1
-    ? `/vms/${vms[0]}`
+    ? `vms/${vms[0]}`
```

Comment:
is it intended to remove the / at the beginning ?

Reply:
No. I fixed it.